libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -40,7 +42,8 @@ class AP_BattMonitor
         BattMonitor_TYPE_ANALOG_VOLTAGE_ONLY        = 3,
         BattMonitor_TYPE_ANALOG_VOLTAGE_AND_CURRENT = 4,
         BattMonitor_TYPE_SMBUS                      = 5,
-        BattMonitor_TYPE_BEBOP                      = 6
+        BattMonitor_TYPE_BEBOP                      = 6,
+        BattMonitor_TYPE_MAXCELL                    = 7
```

Comment:
I think that it is unnecessary if it can distinguish by the value of I 2 C registers.